tests/src/main/java/com/squareup/duktape/tests/OctaneActivity.java

```diff
@@ -0,0 +1,68 @@
+package com.squareup.duktape.tests;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.widget.TextView;
+import com.squareup.duktape.Duktape;
+import java.io.IOException;
+import okio.BufferedSource;
+import okio.Okio;
+
+public final class OctaneActivity extends Activity implements Runnable {
```

Comment:
I was expecting a background thread & a spinner!

If an Android app holds the main thread down for many seconds does the OS give you an "App isn't responding" alert? I know Mac OS X does this...